databroker/eventsource/tests/utils.py

```diff
@@ -11,16 +13,98 @@
 t2 = 1475790875.0159147
 
 def get_config_for_archiver():
-    config = {'name' : 'arch_csx',
-              'url' : arch_csx_url,
-              'timezone' : 'US/Eastern',
-              'pvs' : {'pv1': arch_csx_pv}}
+    config = {'name': 'arch_csx',
+              'url': arch_csx_url,
+              'timezone': 'US/Eastern',
+              'pvs': {'pv1': arch_csx_pv}}
     return config
 
 def get_header_for_archiver():
-    hdr = {'start' : {'time' : t1},
-           'stop' : {'time' :  t2}}
+    hdr = {'start': {'time': t1},
+           'stop': {'time':  t2}}
     return hdr
 
 def build_es_backed_archiver(config):
     return ArchiverEventSource(config)
+
+def get_db_config_for_metadastore():
+    config = {'module': 'databroker.headersource.mongo',
+              'class': 'MDS',
+              'config': {
+                  'host': 'localhost',
+                  'port': '27017',
+                  'database': 'mds',
+                  'timezone' : 'US/Eastern'}}
+    return config
+
+def get_db_config_for_assets():
+    config = {'module': 'databroker.assets.mongo',
+              'class': 'Registry',
+              'config': {
+                  'host': 'localhost',
+                  'port': '27017',
+                  'database': 'fs'}}
+    return config
+
+def get_db_config_for_event_sources():
+	archiver_config =  get_config_for_archiver()
```

Comment:
Something must be off about your editor's configuration. There are couple 5-spaces indents, mixed with 4-spaces indents.